packages/web3-eth/src/index.ts

```diff
@@ -15,6 +15,25 @@ You should have received a copy of the GNU Lesser General Public License
 along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+/**
+ * The `web3-eth` package allows you to interact with an Ethereum blockchain and Ethereum smart contracts.
+ *
+ * To use this package standalone and use its methods use:
+ * ```ts
+ * import { getBalance, getBlockByNumber } from 'web3-eth';
```

Comment:
I think we should be more more descriptive for using such methods, by adding an example here.

Reply:
Addressed by [this](https://github.com/ChainSafe/web3.js/pull/5251/commits/f6033c0bd52381c8532fc4afbc36e265d8b8199c) commit